src/component/SortedLists.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import { FirestoreCollection, withFirestore } from 'react-firestore';
+import dayjs from 'dayjs';
+
+const SortedList = ({ token, handlePurchase, calculateIfPurchased }) => {
+  const concatPath = `/lists/${token}/items`;
+  const now = new Date();
+  const today = dayjs(now);
+
+  return (
+    <section className="listFrame">
+      <h1>&#128722;My Shopping List</h1>
+      {/* could also use &#128717; which is shopping bags */}
+      {/* // items that need to buy soon (fewer than seven days) */}
+      <FirestoreCollection
+        path={concatPath}
+        sort="numberOfDays:asc"
+        filter={['numberOfDays', '<', 7]}
+        render={({ isLoading, data }) => {
+          if (isLoading) {
+            return <div>Still Loading...</div>;
+          } else {
+            return (
+              <div id="soonItems">
+                <h2 className="itemsLabel">Soon Items</h2>
+                <ul>
+                  {data.map(item => (
+                    <li
+                      id={item.id}
+                      key={item.id}
+                      className={calculateIfPurchased(item)}
+                    >
+                      <div
+                        className={item.name}
+                        onClick={handlePurchase}
+                        id={item.id}
+                        aria-label="Soon Item"
+                        aria-required="true"
+                      >
+                        {item.name}
+                      </div>
+                    </li>
+                  ))}
+                </ul>
+              </div>
+            );
+          }
+        }}
+      />
+      {/* // items that need to buy kind of soon (between 7 and 30 days, inclusive)  */}
+      <FirestoreCollection
+        path={concatPath}
+        sort="numberOfDays:asc"
+        filter={[
+          ['numberOfDays', '>=', 7],
+          ['numberOfDays', '<=', 30],
```

Comment:
These should not be inclusive so that the items get added to the correct sub-list.

Reply:
That makes a lot of sense!